ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -65,7 +65,7 @@ images:
       - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.20-openshift-4.15
   to: machine-config-operator
 - dockerfile_literal: |
-    FROM registry.ci.openshift.org/ocp/4.13:rhel-coreos
+    FROM registry.ci.openshift.org/ocp/4.15:rhel-coreos
```

Comment:
Hmm clearly this needs to be auto-updated with branching

Reply:
Under the hood, this is being kept up-to-date within the build context even without this change. I double-checked the image SHA from this image build with the SHA for `registry.ci.openshift.org/ocp/4.15:rhel-coreos` and they were the same. The inline Dockerfile that produces this image could use some work though.